civcraftConfigs/alephconfig.yml

```diff
@@ -173,17 +185,32 @@ monster:
        level: 1
        duration: 1s
        chance: 1.0 
+      blinded:
+       type: BLINDNESS
+       level: 1
+       duration: 1s
+       chance: 1.0 
+      withered:
+       type: WITHER
+       level: 1
+       duration: 2s
+       chance: 1.0 
      blocks_to_spawn_on:
       - STONE
       - DIRT
       - GRASS
       - STATIONARY_LAVA
+      - STATIONARY_WATER 
      blocks_to_spawn_in:
       - AIR
-      - WATER
       - STATIONARY_LAVA
+      - STATIONARY_WATER 
      health: 320
-     chestplate_dropchance: 0.04
+     chestplate_dropchance: 0.02
+     leggins_dropchance: 0.02
+     boots_dropchance: 0.01
+     item_in_dropchance: 0
```

Comment:
Oh, if it was misspelled in the original config, then that explains why the skelly did not drop its chestplate today. Hm, vanilla. What is the correct option? And maybe I'll just let them remain vanilla to reduce issues.